build/debian-iptables/Makefile

```diff
@@ -14,33 +14,28 @@
 
 .PHONY:	build push
 
-REGISTRY?="gcr.io/google_containers"
+REGISTRY?="gcr.io/google-containers"
```

Comment:
It's resolved. We're a ways past supporting a level of Docker that required the underscore.